backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StopVmBackupCommand.java

```diff
@@ -55,6 +60,11 @@ public void init() {
 
     @Override
     protected boolean validate() {
+        List<ImageTransfer> imageTransfers = imageTransferDao.getByBackupId(vmBackup.getId());
+        if (vmBackup.getBackupType() == VmBackupType.Hybrid &&
```

Comment:
Why is this relevant only to hybrid backup?

If you are not sure about the legacy backup flows or don't have time to test them you can leave a TODO here to check the legacy backup.

Testing legacy backup is easy, just disable hybrid backup in engine config.

Reply:
Actually everything's fine with Cold/Live Backup in this case - I actually checked. But then I thought that since we worked only on the Hybrid flow, it would be right not to change only that. But if you think that it's okay then I can remove this condition

Reply:
If we know that the same issue exist in the legacy backup, and this fix handle both,
I don't see a need to limit the fix to the new hybrid backup.